script-library/kubectl-helm-debian.sh

```diff
@@ -140,8 +140,13 @@ fi
 kubectl completion bash > /etc/bash_completion.d/kubectl
 
 # kubectl zsh completion
-mkdir -p /home/${USERNAME}/.oh-my-zsh/completions
-kubectl completion zsh > /home/${USERNAME}/.oh-my-zsh/completions/_kubectl
+if [ ! -z $USERNAME  ]; then
```

Comment:
USERNAME will always be set, but may be root, so this will do what you want I think:
```suggestion
if [ "$USERNAME" != "root"  ]; then
```